<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>93</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PFJ Systems Inc. - Virtual Multimeter VM1</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <widget class="QLCDNumber" name="lcdNumber"/>
   </item>
   <item row="1" column="0">
    <widget class="QRadioButton" name="voltRadio">
     <property name="text">
      <string>Voltage (V)</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QRadioButton" name="currRadio">
     <property name="text">
      <string>Current (A)</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QRadioButton" name="ohmsRadio">
     <property name="text">
      <string>Resistance (Ohms)</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
